store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -2628,6 +2635,7 @@ public static interface ProvisioningValidator {
         static final String RENAME_ACCOUNT_CHECK_DOMAIN_COS_AND_FEATURE =
             "renameAccountCheckDomainCosAndFeature";
         static final String DELETE_ACCOUNT_SUCCEEDED = "deleteAccountSucceeded";
+        static final String IS_VALID_LICENSE = "isValidLicense";
```

Comment:
This is not what we discussed.

Reply:
Made changes as per our last discussion.